UKHomeOffice/cerberus-service
# Cop 9286 total risk score task details

## Description
This PR restructures the Targeting Indicators layout in task details.
JIRA Ticker: https://support.cop.homeoffice.gov.uk/browse/COP-9286

## To Test
- Pull & run against dev
- Confirm that new layout against old layout

> **New layout**
![Targeting Indicators - New Layout](https://user-images.githubusercontent.com/19333750/145439343-bc5d109c-c38e-4d73-88ac-5e7e88c47aa7.png)

vs.

 > **Old layout**
![Targeting Indicators - Old Layout](https://user-images.githubusercontent.com/19333750/145439591-d68bbf33-43b6-4b2a-8663-40d0a33415a8.png)


> Due to the tight coupling of the task details page layout, The new layout will nto exactly match that of the prototype suplied in the Jira ticket [https://support.cop.homeoffice.gov.uk/browse/COP-9286](url).

## Developer Checklist

\* Required
